File: java/OnlyFloat.java
 public void onlyFloat(float a, float b){
	double a = 10.11;
	double b = 4;
	int wholeNumberA = a / 1;
	int wholeNumberB = b / 1;

	if( (a != wholeNumberA) && (b != wholeNumberB)){
		System.out.print("2");}

	else if( (a != wholeNumberA) || (b != wholeNumberB)){
		System.out.print("1");}
	
	else if( (a == wholeNumberA) && (b == wholeNumberB)){
		System.out.print("0");}

}

File: Python/Home_tasks/onlyFloat.py
onlyFloat(a, b){
	a = 10.11
	b = 4
	whole_number_a = a / 1
	whole_number_b = b / 1

	if( (a != whole_number_a) && (b != whole_number_b)):
		System.out.print("2")

	else if( (a != whole_number_a) || (b != whole_number_b)):
		System.out.print("1")
	
	else if( (a == whole_number_a) && (b == whole_number_b)):
		System.out.print("0")

}